Lombiq.Tests.UI.Samples/Tests/DataBaseSnapshotTests.cs

```diff
@@ -2,39 +2,40 @@
 using Lombiq.Tests.UI.Extensions;
 using Lombiq.Tests.UI.Samples.Constants;
 using Lombiq.Tests.UI.Services;
-using System.IO;
 using System.Threading.Tasks;
 using Xunit;
 using Xunit.Abstractions;
 
 namespace Lombiq.Tests.UI.Samples.Tests;
 
-// We can execute test on already existing databases. To test this, we will take a snapshot of the running application,
-// and use that.
-public class DataBaseSnapshotTests : UITestBase
+// We can execute tests on already existing databases. To demo this, we will take a snapshot of the running application,
+// and use that, because we don't have a pre-beaked database. Normally this feature is used to run tests when there is
+// an already existing database, so you don't need to take a snapshot before using the
+// "ExecuteTestFromExistingDBAsync()" method.
+public class DatabaseSnapshotTests : UITestBase
 {
-    public DataBaseSnapshotTests(ITestOutputHelper testOutputHelper)
+    public DatabaseSnapshotTests(ITestOutputHelper testOutputHelper)
         : base(testOutputHelper)
     {
     }
 
     // Here, we set up the application, then we take a snapshot of it, then we use the
-    // "ExecuteTestFromExistingDBAsync()" to run the test on that. Then we test the basic Orchard features to check that
-    // the application was set up correctly.
+    // "ExecuteTestFromExistingDBAsync()" to run the test on that. Finally, we test the basic Orchard features to check
+    // that the application was set up correctly.
     [Theory, Chrome]
-    public Task BasicOrchardFeaturesShouldWorkWithExistingDBSetup(Browser browser) =>
+    public Task BasicOrchardFeaturesShouldWorkWithExistingDatabaseSetup(Browser browser) =>
          ExecuteTestAsync(
                 async context =>
                 {
-                    const string AppForDataBaseTestFolder = "AppForDataBaseTest";
+                    const string AppForDatabaseTestFolder = "AppForDatabaseTest";
 
                     await context.GoToSetupPageAndSetupOrchardCoreAsync(RecipeIds.BasicOrchardFeaturesTests);
-                    await context.Application.TakeSnapshotAsync(AppForDataBaseTestFolder);
+                    await context.Application.TakeSnapshotAsync(AppForDatabaseTestFolder);
 
                     await ExecuteTestFromExistingDBAsync(
                          async context => await context.TestBasicOrchardFeaturesExceptSetupAsync(),
                          browser,
-                         Directory.GetCurrentDirectory() + "//" + AppForDataBaseTestFolder);
+                         AppForDatabaseTestFolder);
```

Comment:
Does this work the same as `Directory.GetCurrentDirectory() + "//" + AppForDataBaseTestFolder` before though? I'm not saying you can just use `AppForDatabaseTestFolder` here without any other changes. I'm saying, the API of `ExecuteTestFromExistingDBAsync` should allow using this relative path, instead of having to prefix is.

Reply:
Yes, because it operates in the same directory, so `Directory.GetCurrentDirectory()` was not necessary. A good example for this is:
https://github.com/Lombiq/UI-Testing-Toolbox/blob/dev/Lombiq.Tests.UI/OrchardCoreUITestBase.cs#L158-L160

Where `appFolder` is not a path just the name of the folder.

Reply:
OK then.